client/layout/masterbar/style.scss

```diff
@@ -86,6 +86,28 @@ $autobar-height: 20px;
 	.tld {
 		color: rgba(255, 255, 255, 0.6);
 	}
+
+	&.is-loggedout {
+		.gridicon {
+			margin: 0;
+		}
+	}
+}
+
+.masterbar__login-links {
+	display: flex;
+	margin-left: auto;
+	margin-right: 12px;
+}
+
+.masterbar__login-links .masterbar__item-content {
```

Comment:
> Do we need the increased specificity?

I'm not sure how that could be avoided — advice welcome. I've moved this to #4931 but have had to keep the specificity.